.infra/Pulumi.prod.yaml

```diff
@@ -62,6 +62,12 @@ config:
       secure: AAABAAAcNFaMFyerA4UVMKYeQYf69uQ0mMfl2I0DIiKOBa6HwtbnTXoA1N4OwidUgnIRlGzdYrgQWzQTMsimcE/Yarpceg==
     magniOrigin:
       secure: AAABAHxH1dQ0e4nHwnseCd65qMd7qz3vSsKV/H0vMQIKWjxS4H7BemSLMpPGEH7YsDLUM9hEMw==
+    meiliIndex:
+      secure: AAABADypa9PiyaCl1GQdpBI+5tcJRA0G894+ZL/lqfW3iWEetKlYQg==
```

Comment:
I think we can manage indexes as enums right? If we have more in the future and they are not exactly secret?

Reply:
Potentially yes, for now we only have one just added one more later of security 😂